src/ArgentPonyWarcraftClient/Properties/AssemblyInfo.cs

```diff
@@ -1,3 +1,4 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("ArgentPonyWarcraftClient.Extensions.DependencyInjection.Tests")]
+[assembly: InternalsVisibleTo("ArgentPonyWarcraftClient.Integration.Tests")]
```

Comment:
Added this so that I didn't have to rehash all of the OAuth token stuff for the `RawBlizzardClient`. Happy to remove it and do that instead if that's desired though.

Reply:
Feel free to be liberal with the `InternalsVisibleTo`.